mobile_config/src/gateway_info.rs

```diff
@@ -103,8 +103,10 @@ pub struct GatewayInfo {
     pub address: PublicKeyBinary,
     pub metadata: Option<GatewayMetadata>,
     pub device_type: DeviceType,
-    // None for V1
+    // None for GatewayInfoProto (V1)
+    pub updated_at: Option<DateTime<Utc>>,
     pub created_at: Option<DateTime<Utc>>,
+    pub refreshed_at: Option<DateTime<Utc>>,
```

Comment:
can you elaborate on the difference between `updated_at` and `refreshed_at` -- perhaps consider adding as comments as well.

Reply:
That's where i got confused - the proto only includes `updated_at` but not `refreshed_at`
https://github.com/helium/proto/blob/b5564eb32d1fb11d7a0570a1c2311b9ed39d297e/src/service/mobile_config.proto#L81-L84

So `refreshed_at` is only related to bookkeeping/internals of this service?

Reply:
> These should be documented in the proto definition

There is no `refreshed_at` in proto definition it is internal variable. 
`updated_at` is documented in proto

> So `refreshed_at` is only related to bookkeeping/internals of this service?

Yes `refreshed_at` is internal thing. I intentionally left only `updated_at` to not confuse clients.
Also I don't see any purposes for clients to have `refreshed_at` field 

Added comments co code.